src/tracks/Pattern.cpp

```diff
@@ -1149,7 +1149,7 @@ void PatternView::paintEvent( QPaintEvent * )
 			if( len < 0 )
 			{
 				p.drawPixmap( x, y, stepoff );
-				p.setOpacity( vol / 100.0 );
+				p.setOpacity( min(vol, 100) / 100.0 );
```

Comment:
I do not get this, there is no way to distinguish notes over 100% velocity now:
![screenshot from 2016-08-12 00 04 40](https://cloud.githubusercontent.com/assets/6282045/17606568/692b46f8-6020-11e6-96ea-b506296792eb.png)

Here's a pattern opened in pianoroll, as you see all the steps look the same while they are not the same velocity.